frontend/src/components/6.Swiping/SwipingPage.js

```diff
@@ -51,29 +54,60 @@ function SwipingPage(props) {
   };
 
   /**
-   * @param {number} index
-   * @return {void}
+   * Swipes card right when clicking yes
    */
   function clickedYes() {
-    if (vote === undefined) {
-      SocketEvents.vote(socketContext.socket, socketContext.code, {
-        name: Data.name,
-        location: Data.location,
-        coords: Data.coords,
-        price: Data.price,
-        rating: Data.rating,
-        images: Data.images,
-      });
-      setVote(true)
+    if(swiperRef.current) {
+      swiperRef.current.swipe('right');
     }
   }
 
   /**
-   *
+   * Swipes card left when clicking no
    */
   function clickedNo() {
-    if (vote === undefined) {
-      setVote(false)
+    if(swiperRef.current) {
+      swiperRef.current.swipe('left');
+    }
+  }
+  
+  /**
+   * This is required as handleCardLeftScreen is called much later than 
+   * when the swipe event is raised. 
+   * @param {*} direction 
+   */
+  function handleSwipe() {
+    // hide vote buttons when swiped
+    setShowVoteButtons(false);
+  }
+
+  /**
+   * Handle a swipe of a card to a specific direction.
+   * If left, then vote is set to false
+   * If right, then vote is set to right and send to server.
+   * If already voted, then do nothing
+   * @param {*} direction 
+   */
+  function handleCardLeftScreen(direction) {
+    if(vote !== undefined) {
+      return;
+    }
+
+    switch(direction) {
+      case 'left':
+        setVote(false);
+        break;
+      case 'right':
+        SocketEvents.vote(socketContext.socket, socketContext.code, {
+          name: Data.name,
+          location: Data.location,
+          coords: Data.coords,
+          cuisine: Data.cuisine,
```

Comment:
This line isn't necessary, now that we have removed everything related to cuisine

Reply:
removed